### ISSUE TITLE
Problem: UUIDs not assigned to directories for non-standard transfer types

### ISSUE THREAD

**Issue by kerim1** [artefactual/archivematica#742]
We are testing the new microservice 'Assign UUIDs to directories' with the latest code changes from the qa/1.0 branch. However, even though we configured the default processing configuration to assign UUIDs to directories, we don't see this microservice called during transfer. I assume this job should have been called right before or right after the job 'Assign file UUIDs to objects' was called, but all I see in the dashboard for 'Approve transfer' is this:
```
Microservice: Approve transfer
- Job Include default Transfer processingMCP.xml 
- Job Verify bag, and restructure for compliance 
- Job Assign checksums and file sizes to objects 
- Job Assign file UUIDs to objects 
- Job Rename with transfer UUID 
- Job Extract zipped bag transfer
- Job Approve zipped bagit transfer
```
Finally, when we obtain the METS from the AIP, we only see UUIDs for files. So no UUIDs for directories here in the METS either.

**Comment by jrwdunham** [comment]
I have confirmed this issue. There are 5 transfer types, each with their own workflow paths and each has its own 'Assign file UUIDs to objects' micro-service. However, only the 'Standard' transfer type path has the new 'Assign UUIDs to directories?' and 'Assign UUIDs to directories' micro-services. A new migration needs to be added so that these micro-services are executed for the remaining 4 transfer types.

The challenge will be ensuring that a single processing config decision (i.e., "Assign UUIDs to directories?") will correctly control 5 distinct workflow paths.

**Comment by jrwdunham** [comment]
This should be fixed by https://github.com/artefactual/archivematica/pull/761. Awaiting CR before merge.

**Comment by sevein** [comment]
@jrwdunham shouldn't this be added to the 1.7 project?

**Comment by jrwdunham** [comment]
Closed by #761

**Comment by currmie** [comment]
Fix verified for Zipped bag and DSpace types of transfer on Ubuntu and CentOS test server deployments.
